### PR TITLE
Prepare 0.3.0 release #docgen

### PR DESCRIPTION
**Pull request by benasher44**
## Changes in this pull request

This prepares the repo for the 0.3.0 release

### Checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I added an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
- [x] I have reviewed the [contributing guide](https://github.com/plangrid/ReactiveLists/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by PlanBot-iOS** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Docs are regenerated when creating new releases.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Docs are generated by using <a href="https://github.com/realm/jazzy">Jazzy</a>. If you want to contribute, please update the <a href="https://github.com/plangrid/ReactiveLists/tree/master/Guides">markdown guides</a> or doc comments.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by benasher44** [comment]
@jessesquires am I breaking the rules by just adding `#docgen` to the title 😬

File: docs/undocumented.json [hunk 1]
```diff
@@ -1,82 +1,12 @@
 {
   "warnings": [
     {
-      "file": "/Users/jesse/GitHub/ReactiveLists/Sources/Diffing.swift",
+      "file": "/Users/basher/Code/ReactiveLists/Sources/Diffing.swift",
       "line": 51,
       "symbol": "AnyDiffableViewModel.model",
       "symbol_kind": "source.lang.swift.decl.var.instance",
       "warning": "undocumented"
```
Comment (jessesquires) [inline docs/undocumented.json]: looks like one more symbol needs docs or `:nodoc:`

Comment (benasher44) [reply docs/undocumented.json]: Fixed!

**Pull request merged**